com.bixense.PasswordCalculator.json

```diff
@@ -0,0 +1,60 @@
+{
+	"app-id": "com.bixense.PasswordCalculator",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.22",
+	"sdk": "org.gnome.Sdk",
+	"command": "pwcalculator",
+	"finish-args": [
+		"--socket=x11",
+		"--socket=wayland"
+	],
+	"cleanup": [ "/include", "/bin/wx*" ],
+	"modules": [
+		{
+			"name": "wxWidgets",
+			"config-opts": "--with-opengl --with-libjpeg --disable-shared --enable-monolithic--with-libtiff --with-libpng --with-zlib --disable-sdltest --enable-unicode --enable-display --enable-propgrid --disable-webkit --disable-webview --disable-webviewwebkit --with-libiconv=/usr",
+			"cxxflags": [
+				"-std=c++11"
+			],
+			"make-args": [
+				"-j16"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets-3.1.0.tar.bz2",
+					"sha256": "e082460fb6bf14b7dd6e8ac142598d1d3d0b08a7b5ba402fdbf8711da7e66da8"
+				}
+			]
+		},
+		{
+			"name": "boost",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.bz2",
+					"sha256": "7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332"
+				}
+			],
+			"build-commands": [
+				"./bootstrap.sh",
+				"./b2 headers",
+				"cp -r boost /app/include/boost"
+			]
+		},
+		{
+			"name": "PasswordCalculator",
+			"buildsystem": "simple",
+			"build-commands": [
+				"./waf configure install --release --prefix=/app"
+			],
+			"sources": [
+				{
+					"type": "git",
```

Comment:
Applications hosted on Flathub should specify a git tag, or at least a specific commit and the sha256 hash, so the builds are reproducible and and more or less stable.